Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -608,3 +607,24 @@ void HexEditorWidget::drop_event(GUI::DropEvent& event)
         open_file(response.value().filename(), response.value().release_stream());
     }
 }
+
+DeprecatedString HexEditorWidget::make_display_string(DeprecatedString const& string)
+{
+    // FIXME: We probably want to replace all control characters here
+    StringBuilder builder;
+    for (auto c : string) {
+        if (c == '\t')
+            builder.append("␉"sv);
+        else if (c == '\n')
+            builder.append("␊"sv);
+        else if (c == '\v')
+            builder.append("␋"sv);
+        else if (c == '\f')
+            builder.append("␌"sv);
+        else if (c == '\r')
```

Comment:
itʼd be good to add `\a` (alarm) and `\b` (backspace) here—also `\e` could be a thing too—not sure if you have the replacement chars for those, though. 🫣

Reply:
I'd like to hold off for this PR, would like to keep it narrow

Reply:
consider that as resolved then! 🙇🏻‍♂️